benchmarks/bench.cpp

```diff
@@ -28,8 +28,9 @@ std::string url_examples[] = {
     "https://www.reddit.com/?after=t3_zvz1ze",
     "https://www.reddit.com/login/?dest=https%3A%2F%2Fwww.reddit.com%2F",
     "postgresql://other:9818274x1!!@localhost:5432/otherdb?connect_timeout=10&application_name=myapp",
-    "http://192.168.1.1",
-    "http://[2606:4700:4700::1111]"};
+    "http://192.168.1.1", // ipv4
```

Comment:
We are changing the benchmark. It makes tracking the performance over time difficult. It might be time to add *new* benchmarks. We can add new files like bench.cpp to benchmark specific cases.

Reply:
I was thinking about the same thing. Either we can separate them according to the host types (ipv4, ipv6, opaque etc.) or according to different contexts (For example: CURL benchmarks etc.)